creationOfJudokaServerSide.gs

```diff
@@ -0,0 +1,81 @@
+const ui = SpreadsheetApp.getUi();
+
+const first_row_index = 2;
+const first_col_index = 1;
+
+const table_person = "Person";
+
+var sheets = SpreadsheetApp.getActiveSpreadsheet();
+const sheet_pers = sheets.getSheetByName(table_person);
+
+const default_img_link = "https://img.freepik.com/vecteurs-premium/portrait-jeune-homme-barbe-coiffure-avatar-masculin-illustration-vectorielle_266660-423.jpg?w=740";
+
+/**
+ * Creates a custom menu in Google Sheets when the spreadsheet opens.
+ */
+function onOpen() {
```

Comment:
It seems that you used this method several times. You could create it once and use it everywhere. Don't copy/paste code when you can simply reuse it.

Reply:
Don't design the code for the sole purpose of showing the code. Design it for your technical needs.